packages/docusaurus-remark-plugin-npm2yarn/src/__tests__/fixtures/import-tabs.md

```diff
@@ -0,0 +1,7 @@
+import Tabs from '@theme/Tabs';
```

Comment:
will it work if you use npm2yarn above the imports?

I think it will but I'd suggest adding usage of npm2yarn above/under the imports just to be 100% sure :)

Reply:
Yes, done it.